.circleci/config.yml

```diff
@@ -274,7 +274,11 @@ step-maybe-electron-dist-strip: &step-maybe-electron-dist-strip
     command: |
       if [ "$STRIP_BINARIES" == "true" ] && [ "`uname`" != "Darwin" ]; then
         cd src
-        electron/script/strip-binaries.py --target-cpu="$TARGET_ARCH"
+        mkdir out/Default/debug
+        # Generate electron.debug. Required to do it from there in order to maintain debug references.
+        cd out/Default/debug
+        ../../../electron/script/strip-binaries.py --target-cpu="$TARGET_ARCH" --compress
+        cd ../../..
```

Comment:
`cd -`

Reply:
no need for `cd` back, the working directory is reset between steps.

Reply:
Done